source/introduction.rst

```diff
@@ -1,14 +1,39 @@
 Introduction
 ************
 
-Learning to program requires much practice. However it is also very rewarding and creative.
+Learning to program requires practice but is also very rewarding and creative.
 
 The purpose of this course it to give you enough programming vocabulary to
-get a taste of what it is.
+get a taste of what it is and to lay the foundational knowledge for you to build upon in the future.
 
-print('Hello')
+
+Python
+======
+
+This introductory course will use a programming language called |python_website|.
+
+.. |python_website| raw:: html
+
+    <a href="https://www.python.org/" target="_blank">Python</a>
+
+.. image:: /images/python-logo.png
+
+Python is popular both in academia, science and other industries.
```

Comment:
Not essential but it might be cool to add a couple ofconcrete example of a large companies or projects that uses python. I know Google and Dropbox use python (at least for some stuff)